scripts/npm-release.js

```diff
@@ -114,7 +114,7 @@ const injectVersions = (packagesToRelease, versions, packages) => {
 
     if (packageJson.dependencies) {
       for (const dependency in packageJson.dependencies) {
-        if (packageJson.dependencies[dependency] === '*') {
+        if (packageJson.dependencies[dependency] === '0.0.0-development' || packageJson.dependencies[dependency] === '*') {
```

Comment:
Why aren't we including the devDeps? Sometimes packages publish with these... right?

Reply:
devDeps doesn't matter for installing from npm, peerDependencies do matter though and we should always put the explicit version number range (even for local packages) directly in the `package.json` (I should probably document this somewhere)